features/spectral_rules/query-parameter-name-is-camel-case.feature

```diff
@@ -0,0 +1,81 @@
+# language: nl
+Functionaliteit: Een query parameter name is in Camel case
+
+    Achtergrond:
+        Gegeven de spectral rule 'query-parameter-names-camel-case'
+
+    Abstract Scenario: Name property van query parameter component is Camel case
+        Gegeven de OpenAPI specificatie
+        """
+        openapi: 3.0.3
+        components:
+          parameters:
+            parameter1:
+              name: <parameter name>
+              in: query
+        """
+        Als de OpenAPI specificatie is gevalideerd met spectral
+        Dan bevat spectral's resultaat geen validatie meldingen
+
+        Voorbeelden:
```

Comment:
hoe bedoel je deze voorbeelden? Komen niet terug in het scenario in de Als en Dan stappen.

Reply:
of laat maar, zit in de Gegeven